src/components/index.js

```diff
@@ -324,6 +324,12 @@ export default class Rnd extends Component {
   }
 
   render() {
+    const innerStyle = {
+      ...boxStyle,
+      zIndex: this.state.z,
+      ...this.props.disableDragging && { cursor: 'normal' },
```

Comment:
This can't be done without using another variable, inline declaration of keys is not supported afaik. I could either leave it this way or go with your solution, but a bit more verbose:
```
    const cursorStyle = this.props.disableDragging ? { cursor: 'normal' } : {};
    const innerStyle = {
      ...boxStyle,
      zIndex: this.state.z,
      ...cursorStyle,
    };
```